### DISCUSSION TITLE
“We couldn't automatically match your request. Please select the correct match from the list below. ”

### DISCUSSION THREAD

**Discussion by haha6311** [sct/overseerr#2734] [Help] [answered]
![image](https://user-images.githubusercontent.com/36618980/166399813-c60af7df-18fe-461d-baba-4d63095aadc7.png)
“无法自动配對你的请求。请从以下列表中选择正确的媒体项。”
“We couldn't automatically match your request. Please select the correct match from the list below. ”

usually happens in some tv documentaries  distributed by some small televisions in China.

**Comment by Pzoco** [comment]
I have the same error - for me it is this series:
https://www.themoviedb.org/tv/121004-mille?language=en
https://trakt.tv/shows/mille-2009

**Comment by sct** [comment, marked as answer]
This will happen if there is no entry on TVDB for the title. If it does exist on TVDB, you can set the TVDB ID on themoviedb (edit the title there) so Overseerr can find it.
